cockatrice/src/filter_string.h

```diff
@@ -26,7 +27,7 @@ class FilterString
     explicit FilterString(const QString &exp);
     bool check(CardData card)
     {
-        return result(card);
+        return result(std::move(card));
```

Comment:
instead of std::move this should use a CardData reference

Reply:
CardData is a smart pointer anyhow, so Im not sure your saving anything by moving or referencing it.